content/en/platform/corda/1.4/cenm/upgrade-notes.md

```diff
@@ -170,7 +170,7 @@ service.
 Service locations in the Signing Service configuration are provided automatically by the
 Zone Service in CENM 1.3. However, to enable this, the service aliases have strictly defined
 formats. You must update the task configurations to refer to service aliases matching
-these names. The names are specified in [service aliases](signing-service.md#direct-cenm-service-locations).
+these names. The names are specified in [service aliases](../../../../../en/platform/corda/1.4/cenm/signing-service.html#direct-cenm-service-locations).
```

Comment:
the header - direct-cenm-service-locations doesn't exist...

Reply:
deleted the link